odoo/orm/registry.py

```diff
@@ -119,10 +119,30 @@ def __new__(cls, db_name: str):
 
     @classmethod
     @locked
-    def new(cls, db_name: str, force_demo: bool = False, status: None = None, update_module: bool = False) -> Registry:
-        """ Create and return a new registry for the given database name. """
-        if status is not None:
-            warnings.warn("Deprecated since 19.0, do not set status", DeprecationWarning)
+    def new(
+        cls,
+        db_name: str,
+        *,
+        update_module: bool = False,
+        install_modules: Iterable[str] = (),
+        upgrade_modules: Iterable[str] = (),
```

Comment:
```suggestion
        install_modules: Collection[str] = (),
        upgrade_modules: Collection[str] = (),
```
It is already like this in `load_modules`. Then, you don't have to cast to tuple afterwards.